src/resolvers/AssetResolver.js

```diff
@@ -14,7 +14,7 @@ type Request = {
 
 type Options = {
   test?: RegExp,
-  platform: 'ios' | 'android',
+  platform: string,
```

Comment:
why change it to string?

Reply:
Yeah, I had some problems coming from Worker that Flow can't recognize enum in common string. ¯\_(ツ)_/¯

I'm not really satisfied with it, maybe we should investigate it once more. But it might be better to do it in separate PR. What do you think? Maybe I would use some help there.

Reply:
Type for `platform` is defined here: https://github.com/callstack/haul/blob/8f84cbb077b2a17b8ae9a921ef32b4d590444810/src/types.js#L56

Reply:
If it is a string it would be potentially easier to extend haul to handle other platforms. I'd prefer to keep it this way to allow for easier extension.